src/current/_includes/releases/v23.2/v23.2.0.md

```diff
@@ -188,8 +192,8 @@ This section summarizes the most significant user-facing changes in v23.2.0 and
     <!-- Rows for each feature -->
     <tr>
     <td>
-        <p class="feature-summary">Physical Cluster Replication is now available in a public preview</p>
-        <p class="feature-description"><a href="https://www.cockroachlabs.com/docs/v23.2/physical-cluster-replication-overview">Physical Cluster Replication</a> is an asynchonous replication feature that allows your cluster to recover from full-cluster failure with a low RPO and RTO. In 23.2, it is an Enterprise-only Public Preview feature, requiring a CCL license, and only available for self-hosted CockroachDB deployments.</p>
+        <p class="feature-summary">Physical Cluster Replication is now available in Preview</p>
+        <p class="feature-description"><a href="https://www.cockroachlabs.com/docs/v23.2/physical-cluster-replication-overview">Physical Cluster Replication</a> is an asynchonous replication feature that allows your cluster to recover from full-cluster failure with a low RPO and RTO. In 23.2, it is an Enterprise-only Preview feature, requiring a CCL license, and only available for self-hosted CockroachDB deployments.</p>
```

Comment:
```suggestion
        <p class="feature-summary">Physical Cluster Replication is now available in Preview</p>
        <p class="feature-description"><a href="https://www.cockroachlabs.com/docs/v23.2/physical-cluster-replication-overview">Physical Cluster Replication</a> is an asynchonous replication feature that allows your cluster to recover from full-cluster failure with a low RPO and RTO. In 23.2, it is a Preview feature, requiring an Enterprise license, and only available for self-hosted CockroachDB deployments.</p>
```

I'd like to make this edit but I'm not sure about dropping the CCL mention. As-is, "CCL license" seems ambiguous - on our FAQ we refer to "CCL (Free)" vs " CCL (Paid)". Regardless, is it sufficient to mention an "Enterprise license" as we do elsewhere in the docs?

Reply:
@mikeCRL let's make this change as you suggest. The other wording is confusing.